src/wp-includes/style-engine/class-wp-style-engine-processor.php

```diff
@@ -86,19 +86,21 @@ public function add_rules( $css_rules ) {
 	 *
 	 * @since 6.1.0
 	 *
+	 * Since 6.3.0 Optimization is no longer the default.
```

Comment:
This is not a correct `since` mention (should be something like `@since 6.3.0 $optimize now defaults to false.`) but I'm not sure we need a `since` mention at the first place.
Any thought @SergeyBiryukov?

Reply:
I think we should go with something like: @audrasjb 

	 * Optimization is no longer the default
	 * @ticket 56467

Reply:
Thanks for the proposal @Rajinsharwar, but this is not a correct way to use `since` mentions.
Please check Docblock standards: https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#since-section-changelogs

Reply:
Thanks for that @audrasjb, just updated it. Can you take a look?